samples/openthread/cli/README.rst

```diff
@@ -333,11 +333,126 @@ To test the Thread 1.2 features, complete the following steps:
+
+      .. code-block:: console
+
+         uart:~$ ot linkmetrics mgmt fe80:0:0:0:10b1:93ea:c0ee:eeb7 forward 1 dra pqmr
+         Done
+         Received Link Metrics Management Response from: fe80:0:0:0:10b1:93ea:c0ee:eeb7
+         Status: Success
+
+   #. Send an MLE Link Probe message to the peer:
+
+      .. code-block:: console
+
+         uart:~$ ot linkmetrics probe fe80:0:0:0:10b1:93ea:c0ee:eeb7 1 10
+         Done
+
+   #. Perform a Link Metrics query (Forward Tracking Series):
+
+      .. code-block:: console
+
+         uart:~$ ot linkmetrics query fe80:0:0:0:10b1:93ea:c0ee:eeb7 forward 1
+         Done
+         Received Link Metrics Report from: fe80:0:0:0:10b1:93ea:c0ee:eeb7
+         - PDU Counter: 13 (Count/Summation)
+         - LQI: 212 (Exponential Moving Average)
+         - Margin: 60 (dB) (Exponential Moving Average)
+         - RSSI: -40 (dBm) (Exponential Moving Average)
+
+   #. Send a Link Metrics Management Request to register an Enhanced-ACK Based Probing:
+
+      .. code-block:: console
+
+         uart:~$ ot linkmetrics mgmt fe80:0:0:0:10b1:93ea:c0ee:eeb7 enhanced-ack register qm
+         Done
+         Received Link Metrics Management Response from: fe80:0:0:0:10b1:93ea:c0ee:eeb7
+         Status: Success
+
+   #. Send a Link Metrics Management Request to clear an Enhanced-ACK Based Probing:
+
+      .. code-block:: console
+
+         uart:~$ ot linkmetrics mgmt fe80:0:0:0:10b1:93ea:c0ee:eeb7 enhanced-ack clear
+         Done
+         Received Link Metrics Management Response from: fe80:0:0:0:10b1:93ea:c0ee:eeb7
+         Status: Success
+
+#. Verify the CSL functionality:
+
+   a. Send an ICMPv6 Echo Request from the leader kit to the router kit link local address:
+
+      .. code-block:: console
+
+         uart:~$ ot ping fe80:0:0:0:acbd:53bf:1461:a861
+         16 bytes from fe80:0:0:0:acbd:53bf:1461:a861: icmp_seq=2 hlim=64 time=2494ms
+         1 packets transmitted, 1 packets received. Packet loss = 0.0%. Round-trip min/a
+         Done
+
+      As we can observe, there is a long latency on the reply of up to 3000 ms due to the indirect transmission mechanism based on data polling.
+
+   #. Enable CSL Receiver on the router kit (now SED) by configuring a CSL Period of 0.5 seconds:
+
+      .. code-block:: console
+
+         uart:~$ ot csl period 3125
+         Done
+
+   #. Send an ICMPv6 Echo Request from the leader kit to the router kit link local address:
+
+   .. code-block:: console
+      uart:~$ ot ping fe80:0:0:0:acbd:53bf:1461:a861
+      uart:~$ W: TX_STARTED event will be triggered without delay
+      16 bytes from fe80:0:0:0:acbd:53bf:1461:a861: icmp_seq=3 hlim=64 time=421ms
+      1 packets transmitted, 1 packets received. Packet loss = 0.0%. Round-trip min/a
+      Done
+
+      Now the reply latency is reduced to a value below 500 ms since the transmission from the leader is performed via CSL, based on the CSL Information Elements sent by the CSL Receiver.
```

Comment:
```suggestion
      Observe that the reply latency is reduced to a value below 500 ms.
      The reduction occurs because the transmission from the leader is performed via CSL, based on the CSL Information Elements sent by the CSL Receiver.
```